javascript-carousel/main.js

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable no-unused-vars */
+var $circleOne = document.querySelector('.circle-one');
+var $circleTwo = document.querySelector('.circle-two');
+var $circleThree = document.querySelector('.circle-three');
+var $circleFour = document.querySelector('.circle-four');
+var $circleFive = document.querySelector('.circle-five');
+var $img = document.querySelector('img');
+var $rightArrow = document.querySelector('.right');
+var $leftArrow = document.querySelector('.left');
+
+function pictureOne(event) {
+  $img.setAttribute('src', 'images/025.png');
+  $circleOne.className = 'circle circle-one circle-background';
+  $circleTwo.className = 'circle circle-two';
+  $circleThree.className = 'circle circle-three';
+  $circleFour.className = 'circle circle-four';
+  $circleFive.className = 'circle circle-five';
+}
+
+$circleOne.addEventListener('click', pictureOne);
+
+function pictureTwo(event) {
+  $img.setAttribute('src', 'images/039.png');
+  $circleOne.className = 'circle circle-one';
+  $circleTwo.className = 'circle circle-two circle-background';
+  $circleThree.className = 'circle circle-three';
+  $circleFour.className = 'circle circle-four';
+  $circleFive.className = 'circle circle-five';
+
+}
+$circleTwo.addEventListener('click', pictureTwo);
+
+function pictureThree(event) {
+  $img.setAttribute('src', 'images/001.png');
+  $circleOne.className = 'circle circle-one';
+  $circleTwo.className = 'circle circle-two';
+  $circleThree.className = 'circle circle-three circle-background';
+  $circleFour.className = 'circle circle-four';
+  $circleFive.className = 'circle circle-five';
+}
+$circleThree.addEventListener('click', pictureThree);
+
+function pictureFour(event) {
+  $img.setAttribute('src', 'images/004.png');
+  $circleOne.className = 'circle circle-one';
+  $circleTwo.className = 'circle circle-two';
+  $circleThree.className = 'circle circle-three';
+  $circleFour.className = 'circle circle-four circle-background';
+  $circleFive.className = 'circle circle-five';
+}
```

Comment:
This is all very manual. If you want to add additional images, you need to make a new function every time. Instead, I recommend you make an array of all the possible `src` values for the image in the carousel, and then make a variable to keep track of the index for the current image. You should also make use of a loop over the dots to show the one for the current image as solid, and the rest as hollow. I suggest putting a `data-` attribute on each dot icon, maybe something like `data-image-index`